core/trino-parser/src/test/java/io/trino/sql/parser/TestStatementBuilder.java

```diff
@@ -218,6 +218,9 @@ public void testStatementBuilder()
 
         printStatement("alter table a.b.c drop column x");
 
+        printStatement("alter materialized view foo set properties (a='1')");
```

Comment:
brackets should not be needed, see `io/trino/sql/parser/TestStatementBuilder.java:210`